### DISCUSSION TITLE
Welcome email .asc files

### DISCUSSION THREAD

**Discussion by vertigo220** [streetwriters/notesnook#1846] [Q&A]
The email I received when I registered contained a notesnook_pub.asc and signature.asc file. I'm assuming these are just to verify that notesnook is the sender, but I wanted to make sure they aren't something I need to keep, or keep private for that matter. They seem like standard files included in, what I assume, are emails from systems that can use PGP even when the email isn't encrypted, but it seemed strange to be included in a welcome email, so I just wanted to be sure.

**Comment by thecodrr** [comment]
They are intentional and all official emails should include these. This hasn't been documented yet since it is a very new feature but the purpose is clear: give users a way to verify the emails from Notesnook and hence prevent spam.
